axonserver/src/main/java/io/axoniq/axonserver/metric/CounterMetric.java

```diff
@@ -30,22 +30,27 @@ public String getName() {
     }
 
     @Override
-    public long size() {
+    public long value() {
         return valueProvider.get();
     }
 
     @Override
     public long min() {
-        return size();
+        return value();
     }
 
     @Override
     public long max() {
-        return size();
+        return value();
     }
 
     @Override
     public double mean() {
-        return size();
+        return value();
+    }
+
+    @Override
+    public double doubleValue() {
```

Comment:
if the reason to add doubleValue method is that the metric can be represented by a double, why not to generify the number type instead of adding a method?